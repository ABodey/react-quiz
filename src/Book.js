import React, { PureComponent } from 'react'
import PropTypes from 'prop-types'

class Book extends PureComponent {
    static PropTypes ={
        author: PropTypes.string.isRequired,
        title: PropTypes.string.isRequired    
    }

    render() {
        const { author, title } = this.props

        return(
            <li>{`${title} by ${author}`}</li>
        )
    }

}

export default Book
