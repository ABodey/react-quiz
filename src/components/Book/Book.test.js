import React from 'react'
import { shallow } from 'enzyme'
import Book from 'components/Book/Book'

describe('<Book />', () => {
    it ('should match snapshots', () => {
      const wrapper = shallow (<Book />)

      expect(wrapper).toMatchSnapshot()
    })
})

