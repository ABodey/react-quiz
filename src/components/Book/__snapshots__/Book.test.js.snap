// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Book /> should match snapshots 1`] = `
<li>
   
  undefined by undefined
   
</li>
`;
